plugins/robot_windows/robot_programming_benchmark/robot_programming_benchmark.html

```diff
@@ -0,0 +1,101 @@
+        <label class="tab" for="overview">Overview</label>
+        <div class="panel">
+          <p>This benchmark will teach you how to program a robot and run the simulation.</p>
+          <p>The tab entitled <b>Metrics</b> measures your progress on an
+            exercise where you have to program a robot to cover a certain distance.</p>
+          <p>The tab entitled <b>Instructions</b> explains you how to
+            open the source code editor to edit the robot controller program, save it and run the simulation with your modified code.
+          </p>
+        </div>
+        <input class="radio" id="metrics" name="group" type="radio">
+        <label class="tab" for="metrics">Metrics</label>
+        <div class="panel">
+          <div style='text-align:center' title='Your performance increases as you get closer to the target position.'>
+            <span id='achievement'>0%</span></div>
+          <br>
+          <p>Program the Thymio II robot to move straight forward and stop.</p>
+          <p>The performance value increases as the robot gets closer
+           to the target position situated exactly one parquetry square in front of it. It will decrease if the robot continues away from this position.</p>
+           <p>The <b>Instructions</b> tab explains the step-by-step
+           procedure to follow.</p>
+        </div>
+        <input class="radio" id="instructions" name="group" type="radio">
+        <label class="tab" for="instructions">Instructions</label>
+        <div class="panel">
+          <h2>Programming a robot in Python</h2>
+           <ol>
+            <li><b>Open the source code editor</b>: click on the source code editor icon <img src='code_editor.png' alt='&#60; &#62;'>.
+              In the window that was just opened, click on the Explorer icon <img src='explorer.png' alt='explorer'>.
+              Under the WEBOTS-PROJECT tab, click on the "edit_me.py" file.
+              This is a simple robot controller coded in Python.
+            </li>
+            <li><b>Edit the controller</b>: on line 7, modify the value assigned to the <em>target</em> variable: instead of
+             <em>0</em>, try to set a different value, for example 10.</li>
+            <li><b>Save your program</b>: in the source code editor, press <em id='saveShortcut'>Ctrl-S</em> to save your program.
+            </li>
+            <li><b>Reset the simulation</b>: press the reset button
+             <img src='reset.png' alt='reset'> to take into account your modifications.</li>
+            <li><b>Run the simulation</b>: press the play button <img src='real_time.png' alt='real-time'> to run the simulation.
+            </li>
+            <li><b>Observe the result</b>: the robot should move forward. Your performance should be updated in real time on the
+             <b>Metrics</b> tab. After a while, the robot should stop and
+             your performance should be recorded (if you are logged in).
```

Comment:
```suggestion
             your performance should be recorded (if you are running an evaluation action from your GitHub main branch).
```

Reply:
This sentence is only visible to the user at times where the score is not going to be stored to the leaderboard (when trying on webots.cloud or when launched locally). Maybe we should remove the "your performance should be recorded \[...]" part completely?

Reply:
Yes, good point.